File: cpp/4th_week/contohsoal1/MainMatriks.cpp
#include "Matriks.cpp"

int main()
{
    int baris = 0, kolom = 0, temp = 0;
    int i,j;

    cout << "Masukan Baris: ";
    cin >> baris;

    cout << "Masukan Kolom: ";
    cin >> kolom;

    // inisialiasi matriks
    Matriks m(baris, kolom);
    
    // mengisi matriks
    for ( i = 0; i < baris; i++)
    {
        for ( j = 0; j < kolom; j++)
        {
            cout << "Masukan angka:" << endl;
            cin >> temp;
            m.setSel(i, j, temp);
        }
    }

    // menampilkan matriks
    for (i = 0; i < baris; i++) 
    {
        for (j = 0; j < kolom; j++) 
        {
            cout << m.getSel(i, j) << " ";
        }
        cout << "\n";
    }
        
    return 0;
}

File: python/4th_week/contohsoal1/MainMatriks.py
from Matriks import Matriks

baris = int(input("Masukan Baris: "))
kolom = int(input("Masukan Kolom: "))
print(baris)
print(kolom)
m = Matriks(baris, kolom)
# m.setMat()

# mengisi matriks
for i in range(baris):
    print("ini i")
    for j in range(kolom):
        print("ini j")
        temp = int(input("Masukan Angka: "))
        m.setSel(i, j, temp)
    

# menampilkan matriks
for i in range(baris):
    for j in range(kolom):
        print(f"{m.getSel(i, j)} ")
    


